README.md

```
@@ -125,9 +125,11 @@ kfk-gift-registry/
       <img src="https://img.shields.io/badge/💻_engineer-27AE60?style=flat-square"/>
     </td>
     <td align="center" width="160">
-      <img src="assets/team/nadia-meyerovich.jpg" width="110" style="border-radius:10%;"/><br/>
-      <b>Nadia Meyerovich</b><br/>
-      <img src="https://img.shields.io/badge/💻_engineer-27AE60?style=flat-square"/>
+      <a href="https://www.linkedin.com/in/nadia-meyerovich-4a30a9215/" target="_blank" rel="noreferrer noopener">
+        <img src="assets/team/nadia-meyerovich.JPEG" width="110" style="border-radius:10%;"/><br/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)